tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -29,25 +36,50 @@ def generate_chat_history(model_path, device, pipeline_config, questions):
     return chat_history
 
 
+generation_configs = [
+    get_greedy_n_stop_strings_exclude_from_output(),
+    get_greedy_n_stop_strings_include_to_output()
+]
 @pytest.mark.skipif(sys.platform in ["darwin", "linux"], reason="Not supposed to work on mac. Segfault on linux CI")
```

Comment:
please, disable all test on arm64 as well.

BTW, you can skip all tests on module level https://docs.pytest.org/en/stable/how-to/skipping.html#id1 where NPU plugin is not available